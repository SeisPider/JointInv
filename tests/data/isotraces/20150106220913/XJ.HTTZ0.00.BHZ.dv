MFT96 R U T -1 95.256    3.4398    1.0000    6373.2     0.7460E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 93.091    3.4614    1.0000    6373.2     0.7738E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 91.022    3.4822    1.0000    6373.2     0.7918E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 89.043    3.5022    1.0000    6373.2     0.8017E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 87.149    3.5212    1.0000    6373.2     0.8068E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 85.333    3.5394    1.0000    6373.2     0.8125E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 83.592    3.5567    1.0000    6373.2     0.8257E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 81.920    3.5731    1.0000    6373.2     0.8542E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 80.314    3.5886    1.0000    6373.2     0.9050E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 78.769    3.6032    1.0000    6373.2     0.9827E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 77.283    3.6168    1.0000    6373.2     0.1088E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 75.852    3.6296    1.0000    6373.2     0.1218E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 74.473    3.6414    1.0000    6373.2     0.1370E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 73.143    3.6522    1.0000    6373.2     0.1538E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 71.860    3.6622    1.0000    6373.2     0.1717E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 70.621    3.6712    1.0000    6373.2     0.1903E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 69.424    3.6793    1.0000    6373.2     0.2092E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 68.267    3.6865    1.0000    6373.2     0.2281E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 67.148    3.6928    1.0000    6373.2     0.2468E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 66.065    3.6983    1.0000    6373.2     0.2651E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 65.016    3.7028    1.0000    6373.2     0.2829E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 64.000    3.7065    1.0000    6373.2     0.3003E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 63.015    3.7094    1.0000    6373.2     0.3172E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 62.061    3.7114    1.0000    6373.2     0.3339E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 61.134    3.7126    1.0000    6373.2     0.3504E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 60.235    3.7131    1.0000    6373.2     0.3670E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 59.362    3.7127    1.0000    6373.2     0.3839E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 58.514    3.7116    1.0000    6373.2     0.4013E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 57.690    3.7098    1.0000    6373.2     0.4194E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 56.889    3.7073    1.0000    6373.2     0.4383E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 56.110    3.7041    1.0000    6373.2     0.4583E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 55.351    3.7002    1.0000    6373.2     0.4792E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 54.613    3.6957    1.0000    6373.2     0.5011E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 53.895    3.6906    1.0000    6373.2     0.5238E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 53.195    3.6849    1.0000    6373.2     0.5472E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 52.513    3.6786    1.0000    6373.2     0.5712E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 51.848    3.6719    1.0000    6373.2     0.5953E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 51.200    3.6646    1.0000    6373.2     0.6195E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 50.568    3.6568    1.0000    6373.2     0.6434E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 49.951    3.6486    1.0000    6373.2     0.6668E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 49.349    3.6399    1.0000    6373.2     0.6896E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 48.762    3.6308    1.0000    6373.2     0.7116E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 48.188    3.6214    1.0000    6373.2     0.7328E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 47.628    3.6116    1.0000    6373.2     0.7532E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 47.080    3.6015    1.0000    6373.2     0.7730E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 46.545    3.5911    1.0000    6373.2     0.7923E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 46.022    3.5804    1.0000    6373.2     0.8113E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 45.511    3.5694    1.0000    6373.2     0.8303E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 45.011    3.5583    1.0000    6373.2     0.8498E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 44.522    3.5469    1.0000    6373.2     0.8700E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 44.043    3.5353    1.0000    6373.2     0.8913E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 43.574    3.5236    1.0000    6373.2     0.9140E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 43.116    3.5117    1.0000    6373.2     0.9384E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 42.667    3.4997    1.0000    6373.2     0.9648E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 42.227    3.4877    1.0000    6373.2     0.9933E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 41.796    3.4755    1.0000    6373.2     0.1024E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 41.374    3.4633    1.0000    6373.2     0.1057E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 40.960    3.4511    1.0000    6373.2     0.1091E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 40.554    3.4388    1.0000    6373.2     0.1128E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 40.157    3.4266    1.0000    6373.2     0.1166E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 39.767    3.4144    1.0000    6373.2     0.1205E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 39.385    3.4022    1.0000    6373.2     0.1246E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 39.010    3.3900    1.0000    6373.2     0.1286E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 38.642    3.3780    1.0000    6373.2     0.1327E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 38.280    3.3660    1.0000    6373.2     0.1368E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 37.926    3.3542    1.0000    6373.2     0.1408E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 37.578    3.3424    1.0000    6373.2     0.1447E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 37.236    3.3308    1.0000    6373.2     0.1487E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 36.901    3.3193    1.0000    6373.2     0.1525E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 36.571    3.3081    1.0000    6373.2     0.1564E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 36.248    3.2969    1.0000    6373.2     0.1603E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 35.930    3.2860    1.0000    6373.2     0.1643E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 35.617    3.2753    1.0000    6373.2     0.1685E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 35.310    3.2648    1.0000    6373.2     0.1729E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 35.009    3.2545    1.0000    6373.2     0.1776E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 34.712    3.2444    1.0000    6373.2     0.1827E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 34.420    3.2347    1.0000    6373.2     0.1882E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 34.133    3.2251    1.0000    6373.2     0.1941E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 33.851    3.2159    1.0000    6373.2     0.2005E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 33.574    3.2069    1.0000    6373.2     0.2073E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 33.301    3.1982    1.0000    6373.2     0.2145E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 33.032    3.1898    1.0000    6373.2     0.2221E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 32.768    3.1818    1.0000    6373.2     0.2300E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 32.508    3.1741    1.0000    6373.2     0.2380E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 32.252    3.1667    1.0000    6373.2     0.2463E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 32.000    3.1596    1.0000    6373.2     0.2545E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 31.752    3.1530    1.0000    6373.2     0.2627E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 31.508    3.1466    1.0000    6373.2     0.2707E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 31.267    3.1407    1.0000    6373.2     0.2785E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 31.030    3.1352    1.0000    6373.2     0.2859E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 30.797    3.1300    1.0000    6373.2     0.2929E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 30.567    3.1253    1.0000    6373.2     0.2993E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 30.341    3.1210    1.0000    6373.2     0.3052E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 30.118    3.1171    1.0000    6373.2     0.3103E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 29.898    3.1136    1.0000    6373.2     0.3146E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 29.681    3.1106    1.0000    6373.2     0.3180E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 29.468    3.1081    1.0000    6373.2     0.3205E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 29.257    3.1061    1.0000    6373.2     0.3219E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 29.050    3.1045    1.0000    6373.2     0.3221E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 28.845    3.1035    1.0000    6373.2     0.3212E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 28.643    3.1030    1.0000    6373.2     0.3190E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 28.444    3.1030    1.0000    6373.2     0.3157E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 28.248    3.1035    1.0000    6373.2     0.3112E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 28.055    3.1046    1.0000    6373.2     0.3056E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 27.864    3.1063    1.0000    6373.2     0.2992E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 27.676    3.1086    1.0000    6373.2     0.2921E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 27.490    3.1114    1.0000    6373.2     0.2845E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 27.307    3.1149    1.0000    6373.2     0.2767E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 27.126    3.1191    1.0000    6373.2     0.2691E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 26.947    3.1239    1.0000    6373.2     0.2619E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 26.771    3.1294    1.0000    6373.2     0.2553E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 26.597    3.1356    1.0000    6373.2     0.2494E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 26.426    3.1425    1.0000    6373.2     0.2443E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 26.256    3.1502    1.0000    6373.2     0.2398E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 26.089    3.1587    1.0000    6373.2     0.2359E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 25.924    3.1679    1.0000    6373.2     0.2321E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 25.761    3.1781    1.0000    6373.2     0.2281E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 25.600    3.1890    1.0000    6373.2     0.2236E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 25.441    3.2009    1.0000    6373.2     0.2182E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 25.284    3.2137    1.0000    6373.2     0.2116E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 25.129    3.2275    1.0000    6373.2     0.2037E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 95.000    3.4424    1.0000    6373.2     0.7493E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 90.000    3.4925    1.0000    6373.2     0.7969E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 85.000    3.5427    1.0000    6373.2     0.8150E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 80.000    3.5916    1.0000    6373.2     0.9208E-06   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 75.000    3.6368    1.0000    6373.2     0.1312E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 70.000    3.6754    1.0000    6373.2     0.2001E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 65.000    3.7029    1.0000    6373.2     0.2832E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 60.000    3.7130    1.0000    6373.2     0.3715E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 55.000    3.6981    1.0000    6373.2     0.4896E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 50.000    3.6492    1.0000    6373.2     0.6649E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 48.000    3.6181    1.0000    6373.2     0.7396E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 46.000    3.5799    1.0000    6373.2     0.8121E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 44.000    3.5342    1.0000    6373.2     0.8934E-05   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 42.000    3.4813    1.0000    6373.2     0.1009E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 40.000    3.4217    1.0000    6373.2     0.1182E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 38.000    3.3566    1.0000    6373.2     0.1400E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 36.000    3.2884    1.0000    6373.2     0.1634E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 34.000    3.2207    1.0000    6373.2     0.1971E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 32.000    3.1596    1.0000    6373.2     0.2545E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 30.000    3.1152    1.0000    6373.2     0.3126E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 29.000    3.1043    1.0000    6373.2     0.3219E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 28.000    3.1051    1.0000    6373.2     0.3038E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 27.000    3.1225    1.0000    6373.2     0.2640E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
MFT96 R U T -1 26.000    3.1637    1.0000    6373.2     0.2338E-04   55.204899  163.244598   37.160000   79.059998 0  1 COMMENT
