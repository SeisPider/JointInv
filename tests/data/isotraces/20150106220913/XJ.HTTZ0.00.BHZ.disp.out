SURF96 R U T -1 95.000    3.4424    1.0000    
SURF96 R C T -1 95.000    6.0787    1.0000    
SURF96 R U T -1 90.000    3.4925    1.0000    
SURF96 R C T -1 90.000    5.8472    1.0000    
SURF96 R U T -1 85.000    3.5427    1.0000    
SURF96 R C T -1 85.000    5.6395    1.0000    
SURF96 R U T -1 80.000    3.5916    1.0000    
SURF96 R C T -1 80.000    5.4532    1.0000    
SURF96 R U T -1 75.000    3.6368    1.0000    
SURF96 R C T -1 75.000    5.2853    1.0000    
SURF96 R U T -1 70.000    3.6754    1.0000    
SURF96 R C T -1 70.000    5.1330    1.0000    
SURF96 R U T -1 65.000    3.7029    1.0000    
SURF96 R C T -1 65.000    4.9935    1.0000    
SURF96 R U T -1 60.000    3.7130    1.0000    
SURF96 R C T -1 60.000    4.8640    1.0000    
SURF96 R U T -1 55.000    3.6981    1.0000    
SURF96 R C T -1 55.000    4.7408    1.0000    
SURF96 R U T -1 50.000    3.6492    1.0000    
SURF96 R C T -1 50.000    4.6192    1.0000    
SURF96 R U T -1 48.000    3.6181    1.0000    
SURF96 R C T -1 48.000    4.5696    1.0000    
SURF96 R U T -1 46.000    3.5799    1.0000    
SURF96 R C T -1 46.000    4.5189    1.0000    
SURF96 R U T -1 44.000    3.5342    1.0000    
SURF96 R C T -1 44.000    4.4664    1.0000    
SURF96 R U T -1 42.000    3.4813    1.0000    
SURF96 R C T -1 42.000    4.4116    1.0000    
SURF96 R U T -1 40.000    3.4217    1.0000    
SURF96 R C T -1 40.000    4.3539    1.0000    
SURF96 R U T -1 38.000    3.3566    1.0000    
SURF96 R C T -1 38.000    4.2928    1.0000    
SURF96 R U T -1 36.000    3.2884    1.0000    
SURF96 R C T -1 36.000    4.2278    1.0000    
SURF96 R U T -1 34.000    3.2207    1.0000    
SURF96 R C T -1 34.000    4.1586    1.0000    
SURF96 R U T -1 32.000    3.1596    1.0000    
SURF96 R C T -1 32.000    4.0855    1.0000    
SURF96 R U T -1 30.000    3.1152    1.0000    
SURF96 R C T -1 30.000    4.0095    1.0000    
SURF96 R U T -1 29.000    3.1043    1.0000    
SURF96 R C T -1 29.000    3.9711    1.0000    
SURF96 R U T -1 28.000    3.1051    1.0000    
SURF96 R C T -1 28.000    3.9332    1.0000    
SURF96 R U T -1 27.000    3.1225    1.0000    
SURF96 R C T -1 27.000    3.8965    1.0000    
SURF96 R U T -1 26.000    3.1637    1.0000    
SURF96 R C T -1 26.000    3.8621    1.0000    
